README.md

```diff
@@ -0,0 +1,78 @@
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
```

Comment:
Hey 👋  it looks nice having a logo at the top, right? If you want the project to look even better kindly consider adding a working link as the `src`, the image is not showing because it's referencing an unexisting source. You can add an image called `my_logo.png` and reference it like below:
```
 <img src="my_logo.png" alt="logo" width="140" height="auto" />
 ```
I hope this helps 😉